### PR TITLE
test: Fix Keptn installation in integration tests

### PR DESCRIPTION
**Pull request by TannerGabriel**
## This PR

- Updates K3S installation
- Waits for K3S to be ready before continuing with Keptn installation

### CODE REVIEW

**Review by pchila — CHANGES_REQUESTED** [review]
The code looks correct but it seems that the `KUBECONFIG` env var is not set and the default $HOME/.kube/config file is not there or does not contain info about the k3s cluster...

It can be seen in runs like this where `kubectl` tries to connect to `localhost:8080`
https://github.com/keptn-sandbox/keptn-service-template-go/runs/6420183593?check_suite_focus=true

File: .github/workflows/integration_tests.yml [hunk 1]
```diff
@@ -55,6 +71,7 @@ jobs:
         with:
           KEPTN_VERSION: ${{ matrix.keptn-version }}
           KUBECONFIG: /etc/rancher/k3s/k3s.yaml
```
Comment (pchila) [inline .github/workflows/integration_tests.yml:73]: I think this variable is redundant since it is set above for the whole workflow :)

**Review by pchila — APPROVED** [review]
There are still some failures but at least now the wait for k3s to be ready seems ok

File: .github/workflows/integration_tests.yml (hunk 1 above)
Comment (TannerGabriel) [reply .github/workflows/integration_tests.yml]: Nice catch! The other failures still need to be debugged and will be fixed in a separate PR.

**Pull request merged**